packages/engine-formula/src/facade/f-formula.ts

```diff
@@ -26,7 +26,9 @@ import { SetFormulaCalculationNotificationMutation, SetFormulaCalculationStartMu
 export class FFormula extends FBase {
     constructor(
         @ICommandService private readonly _commandService: ICommandService,
-        @IConfigService protected readonly _configService: IConfigService
+        @IConfigService protected readonly _configService: IConfigService,
```

Comment:
FFormula should have `Injector` injected to its constructor, so they dependencies required by its extends can be removed from here.